src/components/paragraph.tsx

```diff
@@ -1,19 +1,18 @@
 // ----- Imports ----- //
 
-import { FC, ReactNode } from 'react';
-import React, { css } from '@emotion/core';
+import React, { FC, ReactNode } from 'react';
+import { css } from '@emotion/core';
 import { body } from '@guardian/src-foundations/typography';
 import { remSpace } from '@guardian/src-foundations';
 
-
 // ----- Component ----- //
 
 interface Props {
     children?: ReactNode;
 }
 
 const styles = css`
-    ${body.medium()}
+    ${body.medium()};
```

Comment:
Doesn't `body.medium` generate a set of CSS properties with trailing `;` for you?